lighthouse-core/audits/seo/font-size.js

```diff
@@ -287,16 +294,20 @@ class FontSize extends Audit {
 
     let explanation;
     if (!passed) {
-      const percentageOfFailingText = parseFloat((100 - percentageOfPassingText).toFixed(2));
-      let disclaimer = '';
+      const percentageOfFailingText = parseFloat((100 - percentageOfPassingText).toFixed(2)) / 100;
```

Comment:
I think want to drop the `.toFixed(2)`, because we want it to be a number, right? (and i18n.js will take care of the formatting to 2 decimal places)